src/snowflake/sqlalchemy/snowdialect.py

```diff
@@ -83,28 +85,27 @@
     "DECIMAL": DECIMAL,
     "DOUBLE": FLOAT,
     "FIXED": DECIMAL,
+    "GEOGRAPHY": GEOGRAPHY,
+    "GEOMETRY": GEOMETRY,
     "FLOAT": FLOAT,
     "INT": INTEGER,
     "INTEGER": INTEGER,
     "NUMBER": _CUSTOM_DECIMAL,
-    # 'OBJECT': ?
+    "OBJECT": OBJECT,
     "REAL": REAL,
     "BYTEINT": SMALLINT,
     "SMALLINT": SMALLINT,
     "STRING": VARCHAR,
     "TEXT": VARCHAR,
     "TIME": TIME,
     "TIMESTAMP": TIMESTAMP,
-    "TIMESTAMP_TZ": TIMESTAMP_TZ,
     "TIMESTAMP_LTZ": TIMESTAMP_LTZ,
     "TIMESTAMP_NTZ": TIMESTAMP_NTZ,
+    "TIMESTAMP_TZ": TIMESTAMP_TZ,
     "TINYINT": SMALLINT,
     "VARBINARY": BINARY,
     "VARCHAR": VARCHAR,
     "VARIANT": VARIANT,
-    "OBJECT": OBJECT,
-    "ARRAY": ARRAY,
-    "GEOGRAPHY": GEOGRAPHY,
```

Comment:
It looks like over time stuff has been added not in alphabetical order so cleaning that up a bit.